instruments/newport/newportesp301.py

```diff
@@ -709,7 +709,7 @@ def home(self):
         )
 
     @home.setter
-    def home(self, newval=0):
+    def home(self, newval):
```

Comment:
Giving the setter a default argument doesn't seem to make any sense to me. How would this be set?